src/components/badges/base.tsx

```diff
@@ -2,26 +2,43 @@ import React, { FC, ReactNode } from "react"
 
 import classNames from "classnames"
 
-import { BadgeSize } from "./types"
+import { BadgeBackground, BadgeHeight, BadgeSize } from "./types"
 import styles from "./badges.module.css"
 
 import Tooltip, { TooltipPosition } from "../tooltip/index"
 
 interface Props {
   size: BadgeSize
+  height: BadgeHeight
```

Comment:
What's the difference between `height` and `size`? From the interface, I have a hard time figuring out which one should I use. Would it be possible to only use one property for sizing?

Reply:
`size` was already used for showing/hiding text in badges. Let's say I only want to show the electric leaf in my badge, I'd set `size="small"`. It was implemented like that before, and it would be a breaking change to rename it. 

So yes, you can use both, or not. Height impacts height only, size impacts whether text is shown.

Reply:
Ah! I thought we'd want to have everything named consistently, and that didn't cross my mind. :+1:

Reply:
Alternatively, always feel free and try to reduce the amount of variants we have with the designer. when interfaces become too complex to understand, it can be a sign for too complex design requirements